#define pg_strdup       _strdup

symbol lookup error: /usr/local/lib/psqlodbca.so: undefined symbol: _strdup
[Debian][psqlodbc Release 16_004]

The Debian package build has the same problem, though:
```
ODBCSYSINI=. ODBCINSTINI=./odbcinst.ini ODBCINI=./odbc.ini ./reset-db < ./sampletables.sql
./reset-db: symbol lookup error: ../.libs/psqlodbcw.so: undefined symbol: _strdup
make[2]: *** [Makefile:32: installcheck] Error 127
```

Full build log: https://pgdgbuild.dus.dg-i.net/job/psqlodbc-binaries/83/architecture=amd64,distribution=sid/console

--- a/psqlodbc.h
+++ b/psqlodbc.h
@@ -80,7 +80,11 @@
 #define pg_malloc	malloc
 #define pg_realloc 	realloc
 #define pg_calloc	calloc
+#ifndef WIN32
+#define pg_strdup	strdup
+#else
 #define pg_strdup	_strdup
+#endif /* WIN32 */
 #define pg_free		free
 #endif /* _MIMALLOC_ */
 

